lib/shared/types.ts

```diff
@@ -673,4 +673,4 @@ export let version: string
 // Unlike node, Deno lacks the necessary APIs to clean up child processes
 // automatically. You must manually call stop() in Deno when you're done
 // using esbuild or Deno will continue running forever.
```

Comment:
Also FYI, the new API has `Deno.ChildProcess#ref` and `Deno.ChildProcess#unref` methods that should let most users avoid calling this API (unless they are running in a test with sanitization, like you are here).

I suggest you call `process.unref()` like you do in Node, and then most users don't have to call this method at all anymore :)

Reply:
Yes, that comment is old. Thanks for mentioning it. I just became aware of `unref` while doing this yesterday and still need to look into it.

Although thinking about the bigger picture, I suppose perhaps this whole `deno.land/x/esbuild` library is unnecessary now that Deno has since added support for npm packages. It feels like I should just deprecate and abandon it, and then tell people to import from `npm:esbuild` instead.

Reply:
That is certainly an option - however in that case a user may still want to support the explicit shutdown of the underlying esbuild process.

Reply:
Yes, that's true. Calling `esbuild.stop()` from `npm:esbuild` currently prints the following:

```
Warning: Not implemented: ChildProcess.prototype.disconnect
```

I'm not sure why though as esbuild never calls `disconnect` and there is no stack trace associated with the warning.

Reply:
Note: Calling `unref` seems to have caused a regression: #3682. I'm removing the call to `unref` to fix the regression: #3685.